package.json

```
@@ -32,7 +32,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/plugin-transform-modules-commonjs": "^7.5.0",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@tidepool/viz": "mdblp/viz#dblp.0.4.2",
+    "@tidepool/viz": "mdblp/viz#dblp.0.4.3-rc1",
```

```suggestion
    "@tidepool/viz": "mdblp/viz#dblp.0.4.3",
```
The release is done.